zipline/data/minute_bars.py

```diff
@@ -48,6 +50,99 @@ class BcolzMinuteWriterColumnMismatch(Exception):
     pass
 
 
+class MinuteBarReader(with_metaclass(ABCMeta)):
+
+    @abstractproperty
+    def last_available_dt(self):
+        """
+        Returns
+        -------
+        dt : pd.Timestamp
+            The last minute for which the reader can provide data.
+        """
+        pass
+
+    @abstractproperty
+    def first_trading_day(self):
+        """
+        Returns
+        -------
+        dt : pd.Timestamp
+            The first trading day (session) for which the reader can provide
+            data.
+        """
+        pass
```

Comment:
It was the intent for all to have `pass`; looks like the got clipped while putting in the docstrings. Will make them all the same.